config/application.properties

```diff
@@ -1,7 +1,8 @@
 #Bridge related settings
 bridge.id=my-bridge
 # uncomment the following line to enable Jaeger tracing, check the documentation how to configure the tracer
-#bridge.tracing=jaeger
+bridge.tracing=jaeger-otel
+bridge.tracing.service-name=strimzi-kafka-bridge
```

Comment:
Shouldn't the above be commented to not have tracing enabled by default? At least it's how works today with the comment above explaining to uncomment to enable tracing.

Reply:
Ah, yes, forgot to change it back ... fixed.